server/command.go

```diff
@@ -15,7 +15,7 @@ import (
 const (
 	starterText            = "###### Mattermost Zoom Plugin - Slash Command Help\n"
 	helpText               = `* |/zoom start| - Start a zoom meeting`
-	oAuthHelpText          = `* |/zoom disconnect| - Disconnect from zoom`
+	oAuthHelpText          = `* |/zoom connect| - Connect to Zoom* |/zoom disconnect| - Disconnect from Zoom`
```

Comment:
These commands should not be in a single line. Revert this to the original format present in `master` branch.